guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -21,5 +21,5 @@ ifndef::satellite[]
 Note that this requirement is for Katello users only.
 endif::[]
 
-* You must configure {Project} to use this IPv4 HTTP proxy server as the default proxy.
+* You must configure {Project} to use this HTTP proxy server as the default proxy supporting both IPv4 and IPv6.
```

Comment:
```suggestion
* You must configure {Project} to use this HTTP proxy server as default proxy supporting both IPv4 and IPv6.
```

Reply:
I think this added part makes the sentence very long and hard to read. Perhaps this is easier?
```suggestion
* You must configure {Project} to use this dual stack (supporting both IPv4 and IPv6) HTTP proxy server as the default proxy.
```